Source/ASDisplayNode.mm

```diff
@@ -442,16 +442,29 @@ - (void)asyncTraitCollectionDidChangeWithPreviousTraitCollection:(ASPrimitiveTra
     if (self.primitiveTraitCollection.userInterfaceStyle != previousTraitCollection.userInterfaceStyle) {
       // When changing between light and dark mode, often the entire node needs to re-render.
       // This change doesn't happen frequently so it's fairly safe to render nodes again
-      if (_loaded(self) && self.isLayerBacked) {
-        // Background colors do not dynamically update for layer backed nodes since they utilize CGColorRef
-        // instead of UIColor. We utilize the _backgroundColor instance variable to track the full dynamic color
-        // and apply any changes here when trait collection updates occur.
-        CGColorRef cgBackgroundColor = _backgroundColor.CGColor;
-        if (!CGColorEqualToColor(_layer.backgroundColor, cgBackgroundColor)) {
-          _layer.backgroundColor = cgBackgroundColor;
+      BOOL needsClippingCornerUpdate = NO;
+      CGFloat cornerRadius = _cornerRadius;
+      ASCornerRoundingType cornerRoundingType = _cornerRoundingType;
+      UIColor *backgroundColor = self.backgroundColor;
+      if (_loaded(self)) {
+        if (self.isLayerBacked) {
+          // Background colors do not dynamically update for layer backed nodes since they utilize CGColorRef
+          // instead of UIColor. We utilize the _backgroundColor instance variable to track the full dynamic color
+          // and apply any changes here when trait collection updates occur.
+          CGColorRef cgBackgroundColor = _backgroundColor.CGColor;
```

Comment:
use should use the new local variable here

Reply:
Done